homeassistant/components/telegram_bot/onlysend.py

```diff
@@ -0,0 +1,26 @@
+"""
+Telegram bot only sender implementation.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/telegram_bot.polling/
```

Comment:
I don't like the name either, was hoping someone to suggest a better one, lol

Going to update the description and make the doc.

Thanks for reviewing!